device/iot-device-samples/android-sample/app/src/main/java/com/iothub/azure/microsoft/com/androidsample/MainActivity.java

```diff
@@ -20,6 +20,9 @@
 public class MainActivity extends AppCompatActivity {
 
     String connString = "[device connection string]";
+    String deviceId = "MyAndroidDevice";
```

Comment:
We use deviceId in all SDK samples for all other languages, e.g. https://github.com/Azure/azure-iot-sdk-c/blob/master/iothub_client/samples/iothub_client_sample_amqp/iothub_client_sample_amqp.c (for C), or https://github.com/Azure/azure-iot-sdk-node/blob/master/device/samples/simple_sample_device.js (for Node), etc. Also, this is the structure that we use at docs.microsoft.com for all the samples for the physical devices (e.g. https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-arduino-huzzah-esp8266-get-started)